imsim/meta_data.py

```diff
@@ -0,0 +1,10 @@
+
+import os
+
+imsim_dir = os.path.split(os.path.realpath(__file__))[0]
+install_dir = os.path.split(imsim_dir)[0]
+
+if 'IMSIM_DATA_DIR' in os.environ: # pragma: no cover
+    data_dir = os.environ['IMSIM_DATA_DIR']
+else:
+    data_dir = os.path.join(install_dir, 'imsim', 'data')
```

Comment:
Could we leave out the symlink for the data directory in the new imsim directory , not make install_dir variable here and then just:

```
else:
     data_dir = os.path.join(imsim_dir, 'data')
```

here?

Or is this tied to either the install system, or trying to be flexible about where we might put the data directory?

Reply:
We need the symlink for it to get installed correctly.  But I think you're right that breaking off the `imsim` part from `imsim_dir` and then adding it back here is unnecessary.  (I just copied what we did for GalSim, but I can't think why that would be necessary.)